<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>home</class>
 <widget class="QMainWindow" name="home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>586</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Analizador lexico y sintactico</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Panama"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Utopia</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Compilador</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bt_lexico">
       <property name="text">
        <string>Analizar Lexico</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_sintactico">
       <property name="text">
        <string>Analisis Sintactico</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Analisis Lexico</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="tx_lexico">
       <property name="placeholderText">
        <string>Analisis de lexemas</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Analisis Sintactico</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="tx_sintactico">
       <property name="placeholderText">
        <string>Analisis de gramatica libre de contexto</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="tx_ingreso">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>321</width>
      <height>331</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ingrese el texto analizay o suba un archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_archivo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Subir archivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_limpiar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>delete.png</normaloff>delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="estado">
   <property name="font">
    <font>
     <family>NanumGothicExtraBold</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
